package/yast2.spec

```diff
@@ -37,8 +37,8 @@ BuildRequires:  rubygem(%{rb_default_ruby_abi}:cfa)
 # for running scripts
 BuildRequires:  rubygem(%{rb_default_ruby_abi}:cheetah)
 # For running RSpec tests during build
-BuildRequires:  rubygem(%{rb_default_ruby_abi}:rspec)
 BuildRequires:  update-desktop-files
+BuildRequires:  rubygem(%{rb_default_ruby_abi}:rspec)
```

Comment:
Oh, I guess that the comment `For running RSpec tests during build` should be also moved above this line.